test/hello_world.2asm.js

```diff
@@ -13,8 +13,8 @@ function asmFunc(global, env, buffer) {
  var Math_abs = global.Math.abs;
  var Math_clz32 = global.Math.clz32;
  function add(x, y) {
-  x = x | 0;
-  y = y | 0;
+  x = (x | 0)
```

Comment:
We took out STAT nodes as an optimization, though. I think we should emit `;`s without needing an AST node for them. It should be easy as discussed before, to just emit one after each element in a block (and loop body, and if arms).